### PR TITLE
fix(app): remove legacy modal content background color override prop

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Fix existing usage of the legacy BaseModal component by reverting a change that defaulted the
background color in a way that doesn't cover existing usage.

re #10742

NOTE: this reversion leaves the `LiquidsLabwareDetailsModal` component with slightly incorrect styling (background color should be darker than it is for the modal contents). This will need to be mended, but will require the component to be refactored so as not to depend on the legacy `BaseModal` component.

Fixes bug in app update flow where the spinner would appear as:

Screen Shot 2022-07-01 at 12 05 31 PM

It now appears as it did before:

Screen Shot 2022-07-01 at 4 30 50 PM

# Changelog

- remove the extra prop `contentBackgroundColor` prop that was added to the component library legacy `BaseModal` component which was causing improper defaulting on existing transparent spinner modals in the app update flow

# Review requests

- download this branch and attempt to install another version of the app by switching release channels, when the download modal appears, click through to update the app, you should see the spinner on a transparent background with the text "download in progress", not the opaque white bar in the screen shot 

# Risk assessment
low

### CODE REVIEW

File: app/src/atoms/Modal/index.tsx [hunk 1]
```diff
@@ -52,7 +51,6 @@ export const Modal = (props: ModalProps): JSX.Element => {
     onClose,
     closeOnOutsideClick,
```
Comment (koji) [inline app/src/atoms/Modal/index.tsx:52]: One quick question, this is optional, it won't have any impact on `release_6.0.0`.
You leave this on purpose, don't you?

**Review by koji — APPROVED** [review]
The changes make sense to me.
I just leave one question on this PR.

File: app/src/atoms/Modal/index.tsx (hunk 1 above)
Comment (b-cooper) [reply app/src/atoms/Modal/index.tsx:52]: That's correct, that was my reasoning.  Though I'd prefer neither of the props were added, I was just trying to make the minimal fix for this bug.

Comment (koji) [reply app/src/atoms/Modal/index.tsx:52]: Thank you!

Comment (koji) [reply app/src/atoms/Modal/index.tsx:52]: @b-cooper 
Sorry, I forgot to switch back to my working branch and added code to this PR. I removed all of them.

**Comment by y3rsh** [comment]
### verified alpha5
image

**Pull request merged**